zeppelin-web-angular/src/app/core/paragraph-base/paragraph-base.ts

```diff
@@ -142,6 +144,8 @@ export abstract class ParagraphBase extends MessageListenersManager {
         }
         this.cdr.markForCheck();
       });
+      this.notebookParagraphCodeEditorComponent.editorSettingTriggerAllowed = true;
```

Comment:
We should handle the case where `this.notebookParagraphCodeEditorComponent` is undefined